tests/test_preprocessing/test_feature_generation/test_one_hot_encoder.py

```diff
@@ -1,13 +1,16 @@
+from guacml.dataset import Dataset
 from guacml.preprocessing.column_analyzer import ColType
 from guacml.preprocessing.feature_generation.one_hot_encoder import OneHotEncoder
 import pandas as pd
 import unittest
+import tests.test_util as test_util
 
 
 class TestOneHotEncoder(unittest.TestCase):
 
     def test_encoding(self):
-        enc = OneHotEncoder()
+        config = test_util.load_config()
+        enc = OneHotEncoder(config['pre_processing'])
```

Comment:
No strong opinion here. Just thought we could have one test config that we use in most places in the tests.